docs/minio-admin-complete-guide.md

```diff
@@ -10,6 +10,7 @@ policy       manage canned policies
 credential   change admin server access and secret keys
 config       manage configuration file
 heal         heal disks, buckets and objects on minio server
+top          command similar to top in Linux
```

Comment:
We need to write a better sentence for this command `command similar to top in Linux` doesn't really convey the right meaning, in-fact it is perhaps even grammatically correct. I think you better review this wording with @abperiasamy 

We can quite a few ways

- provides top like statistics for your Minio process
- top like statistics for Minio
- dynamic real-time view of a Minio server
- real-time system summary of a Minio server